include/openmc/capi.h

```diff
@@ -70,7 +70,8 @@ extern "C" {
   int openmc_next_batch(int* status);
   int openmc_nuclide_name(int index, const char** name);
   int openmc_plot_geometry();
-  int openmc_id_map(const void* slice, int32_t* data_out);
+  int openmc_id_map(const void* slice, int32_t *data_out);
+  int openmc_property_map(const void* slice, double *data_out);
```

Comment:
Your stars keep running away from the type :wink:

Reply:
:disappointed: